day_7/fizzbuzz.js

```diff
@@ -0,0 +1,21 @@
+* For any number that is a multiple of 3, log 'Fizz'
+* For any number that is a multiple of 5, log 'Buzz'
+* For any number that is a multiple of both 3 and 5, log 'FizzBuzz'
+* For all other numbers, log the number.
+
+The output of you program will look something like this:
+```
+=> 1, 2, Fizz, 4, Buzz, Fizz, 7, 8, Fizz, Buzz, 11, Fizz, 13, 14, FizzBuzz, ..., 98, Fizz, Buzz
+```
+// MY CODE BELOW //
+
```

Comment:
Great work taking a pass at this!  I would highly recommend reviewing this challenge again.  I believe in order for any of the iterations to print out `FizzBuzz` you would need lines 18-20 to be the first conditional (if/else statement).  When I tested this code in the console, I didn't see any logs of `FizzBuzz`, only `Fizz` or `Buzz`.